package.json

```diff
@@ -35,13 +36,15 @@
     "eslint": "^2.1.0",
     "eslint-plugin-json": "^1.2.0",
     "eslint-plugin-mocha": "^2.0.0",
-    "eslint-plugin-react": "^3.16.1",
+    "eslint-plugin-react": "^4.1.0",
```

Comment:
Also, it has been setup like this since 2.0.2 https://github.com/godaddy/javascript/commits/master